code/game/objects/structures/tables_racks.dm

```diff
@@ -393,6 +399,7 @@
 			add_logs(user, src, "climbed onto")
 			user.Stun(2)
 			return 1
+			tableclimber = null

```

Comment:
uhhhhhhhh I mean I tested it and it worked just fine. Theres also a return just below it. I wasnt sure. Im a pretty new coder so if its misplaced let me know!